### DISCUSSION TITLE
Autodiscover with exclusion list of repositories

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#14779] [Request Help] [answered]
Looking for guidance on the best way to achieve this.

We are in the process of switching to use autodiscover with Bitbucket Cloud so that all eligible repositories, including any new ones created, are automatically opted in to the renovate service.

When performing a dry-run of this, we have found a large number of repositories that are ineligible because they do not use a support package/dependency manager.  

I'd like to find an elegant way to enable autodiscover, but have a list of repositories that are to be excluded

**Comment by rarkins** [comment, marked as answer]
There is autodiscoverFilter

**Reply by setchy** [reply]
Thanks.
